ClientApp/viking-vault/src/display_user_list/UserData.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import '../components/styles.css';
+import './DisplayUsers.css';
+import DefaultProfilePicture from '../UI/user2.png';
+import CardImg from '../UI/GENERICcard-01.png';
+import { constants } from "../ConstantVariables";
+
+export interface IUserData{
+    id: number;
+    firstName: string;
+    lastName: string;
+    address: string;
+    email: string;
+    pictureLink: string;
+    cardNumber: string;
+    expirationDate: string;
+}
+
+class UserData extends React.Component<IUserData, IUserData>{
+    
+    constructor(props: IUserData)
+    {
+        super(props);
+
+        if(props != null)
+        {
+            this.state = {
+                id: props.id,
+                firstName: props.firstName,
+                lastName: props.lastName,
+                address: props.address,
+                email: props.email,
+                pictureLink: props.pictureLink,
+                cardNumber: props.cardNumber,
+                expirationDate: props.expirationDate
+            };
+        }
+        else
+        {
+            this.state = {
+                id: 0,
+                firstName: "no-data",
+                lastName: "no-data",
+                address: "no-data",
+                email: "no-data",
+                pictureLink: "",
+                cardNumber: "no-data",
+                expirationDate: "no-data"
+            };
+        }        
+    }
+
+    render(){
+        return(
+         
+            <div className = "user-container"> 
+                <div className = "user-container-inside">
+                    <div className = "img-container">
+                         {this.state.pictureLink === "" ? <img src = {DefaultProfilePicture} className = "profile-img"/> : <img src = {this.state.pictureLink} className = {this.state.pictureLink}/> }                 
```

Comment:
ClassName is wrong for the else case. Also you could change only the source like <img src= {this.state.pictureLink? : }